packages/contracts-bedrock/contracts/libraries/Predeploys.sol

```diff
@@ -6,6 +6,12 @@ pragma solidity ^0.8.0;
  * @notice Contains constant addresses for contracts that are pre-deployed to the L2 system.
  */
 library Predeploys {
+
+
+    address internal constant BVM_BIT = 0x4200000000000000000000000000000000000021;
```

Comment:
Is this an address on L2? On L2, `BIT` is the native token, other than an ERC20 token.